l1-contracts/contracts/state-transition/IStateTransitionManager.sol

```diff
@@ -48,8 +55,14 @@ interface IStateTransitionManager {
     /// @notice ValidatorTimelock changed
     event NewValidatorTimelock(address indexed oldValidatorTimelock, address indexed newValidatorTimelock);
 
-    /// @notice InitialCutHash changed
-    event NewInitialCutHash(bytes32 indexed oldInitialCutHash, bytes32 indexed newInitialCutHash);
+    /// @notice chain creation parameters changed
+    event NewChainCreationParams(
```

Comment:
Shall we store `oldInitialCutHash` and `newInitialCutHash`? Should it be indexed?

Reply:
discussed in prviate, not needed